eth_test_parser/src/config.rs

```diff
@@ -1,6 +1,10 @@
-pub(crate) const ETH_TESTS_REPO_URL: &str = "https://github.com/ethereum/tests.git";
+use common::config::MAIN_TEST_DIR;
+
+// The PR <https://github.com/ethereum/tests/pull/1380> moved all test versions prior Cancun HF
+// to the `LegacyTests` folder instead.
+pub(crate) const ETH_TESTS_REPO_URL: &str = "https://github.com/ethereum/legacytests.git";
 pub(crate) const ETH_TESTS_REPO_LOCAL_PATH: &str = "eth_tests";
-pub(crate) const GENERAL_GROUP: &str = "BlockchainTests";
+pub(crate) const GENERAL_GROUP: &str = MAIN_TEST_DIR;
 pub(crate) const TEST_GROUPS: [&str; 1] = ["GeneralStateTests"];
 // The following subgroups contain subfolders unlike the other test folders.
-pub(crate) const SPECIAL_TEST_SUBGROUPS: [&str; 3] = ["Cancun", "Shanghai", "VMTests"];
+pub(crate) const SPECIAL_TEST_SUBGROUPS: [&str; 2] = ["Shanghai", "VMTests"];
```

Comment:
There are actually no `_Shanghai` tests in the `Cancun` folder so it can be simply removed.